learning_wavelets/evaluate/utils.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
```

Comment:
Actually I just noticed that this whole file already exists under the name `learning_wavelets/evaluate.py`.

I think you should rather use directly the content of this file.

Also this means you need to rename the folder evaluate to something like `evaluation_scripts`.

Reply:
I noticed that there's a problem, because some functions use cropping with `_tf_crop`. Also, the dictionary of metric functions `METRICS_FUNC` has two elements instead of three (I added the loss based on 'mse')